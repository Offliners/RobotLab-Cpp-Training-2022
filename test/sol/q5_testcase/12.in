16+17-10+5=/17/3%2=+(10*8)*4-(11-14)%6=/((7+7/19)-12)+(17+12+(16-9+9))%14=/7+11+11*18%3=*9-10*17%18=/14+(15-4)-5/6*16/14-18=-4+20%14=*18+17%9=*(8*8)-8*12=
