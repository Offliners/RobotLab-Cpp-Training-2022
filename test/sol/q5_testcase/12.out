28
0
317
-61
2
10
-7
-5
-82
-5344
